pkg/fanal/artifact/image/image_test.go

```diff
@@ -40,8 +40,11 @@ func TestArtifact_Inspect(t *testing.T) {
 			Version:    "3.2.0-r3",
 			SrcName:    "alpine-baselayout",
 			SrcVersion: "3.2.0-r3",
-			Licenses:   []string{"GPL-2.0"},
-			Digest:     "sha1:8f373f5b329c3aaf136eb30c63a387661ee0f3d0",
+			Identifier: types.PkgIdentifier{
+				PURL: "pkg:apk/alpine-baselayout@3.2.0-r3?arch=x86_64",
```

Comment:
Use the correct purl row here.

Reply:
same for all inputs in this test file.

Reply:
done at https://github.com/aquasecurity/trivy/pull/5439/commits/19fbf7b6f7d528e44d1cf720f78727c2956b4e27